src/redux/sagas/actions/initiateSafeTransaction.ts

```diff
@@ -192,11 +197,30 @@ function* initiateSafeTransactionAction({
           id: txHash,
           title,
           safe,
-          transactions,
         },
       },
     });
 
+    /*
+     * Create individual safe transaction data records
+     */
+    for (const transaction of transactions) {
+      yield apolloClient.mutate<
+        CreateSafeTransactionDataMutation,
+        CreateSafeTransactionDataMutationVariables
+      >({
+        mutation: CreateSafeTransactionDataDocument,
+        variables: {
+          input: {
+            ...omit(transaction, 'token'),
+            tokenAddress: transaction.token?.tokenAddress,
+            safeTransactionTransactionsId:
```

Comment:
I feel like this name could be better, how about just `safeTransactionId`? or perhaps `transactionHash` considering that if this is inside a `SafeTransactionData` model then we could conclude that it's a safe transaction so there wouldn't be a need to be so specific.

Reply:
sounds good I was struggling with naming this so I just fell back on amplify's auto naming. Thanks for the suggestions pana!